src/main/java/com/box/sdk/BoxUser.java

```diff
@@ -719,6 +719,14 @@ public Info() {
             super();
         }
 
+        /**
+         * Constructs an Info object by parsing information from a JSON string.
+         * @param  json the JSON string to parse.
+         */
+        public Info(String json) {
+            super(json);
+        }
+
```

Comment:
Can you add this to the other two classes that were missing this constructor, too?
BoxFileUploadSession.Info
BoxWebHook.Info